src/posix.mak

```diff
@@ -280,6 +281,8 @@ TK_SRC = \
 	$(TK)/filespec.h $(TK)/mem.h $(TK)/list.h $(TK)/vec.h \
 	$(TK)/filespec.c $(TK)/mem.c $(TK)/vec.c $(TK)/list.c
 
+STRING_IMPORT_FILES = verstr.h SYSCONFDIR
```

Comment:
I mean you've got a file named `SYSCONFDIR` and a make variable called `SYSCONFDIR` and it's a little confusing.